tests/parametric/conftest.py

```diff
@@ -108,13 +108,13 @@ def python_library_factory() -> APMLibraryTestServer:
         container_img="""
 FROM ghcr.io/datadog/dd-trace-py/testrunner:7ce49bd78b0d510766fc5db12756a8840724febc
```

Comment:
We will need to change this once after the new `testrunner` image is built and uploaded, so code below would work

Reply:
which testrunner image ?